src/vsr/replica_test.zig

```diff
@@ -3,7 +3,7 @@ const assert = std.debug.assert;
 const maybe = stdx.maybe;
 const log = std.log.scoped(.test_replica);
 const expectEqual = std.testing.expectEqual;
-const expect = std.testing.expectEqual;
+const expect = std.testing.expect;
```

Comment:
This one made me soo confused. Reminded me of this today's post https://avva.livejournal.com/3669085.html (might need google translate, but the code is self-explanatory)